src/ui/Pricing/SubscriptionSettings/index.svelte

```diff
@@ -179,12 +179,65 @@
     gap: 16px;
   }
 
+  .overview {
+    margin-top: 6px;
+  }
```

Comment:
Current `.info` and `.overview` referrer to the logically connected elements, hence its behaviour can be defined with a single class `.description`